slides/ds101-worskhop.mdx

```diff
@@ -428,9 +428,19 @@ to find out the relevant parts of your design system.
 
 ---
 
-- [Carbon design system](https://www.carbondesignsystem.com)
-- [Polaris](https://polaris.shopify.com/)
-- [Zendesk Garden](https://garden.zendesk.com/)
+<Figure
+  src="/assets/parts-carbon.png"
```

Comment:
`alt` is missing and it is required. Please add it.

Reply:
Added